tests/Unit/Evolution/Systems/Cce/Equations.py

```diff
@@ -0,0 +1,205 @@
+                  / (4.0 * k) \
+                + 1.0 / 2.0 * eth_jbar * (eth_beta + j * ethbar_j_jbar \
+                                 / (4.0 * k * k_squared) \
+                               - ethbar_j * (-1.0 + 2.0 * k_squared) \
+                                 / (4.0 * k * k_squared) - 1.0 / 2.0 * q)
+
+    dy_jbar = np.conj(dy_j)
+    dy_j_jbar = j * dy_jbar + jbar * dy_j
+    eth_dy_jbar = np.conj(ethbar_dy_j)
+    eth_j_dy_jbar = np.conj(ethbar_jbar_dy_j)
+    ubar = np.conj(u)
+
+    script_bj = (- eth_u * jbar * dy_j_jbar / (4.0 * k) + 1.0 / 2.0 * dy_w \
+                 + 1.0 / (4.0 * r) + 1.0 / 4.0 * ethbar_j * dy_jbar * u \
+                 - ethbar_j_jbar * dy_j_jbar * u / (8.0 * k_squared) \
+                 - 1.0 / 4.0 * j * eth_dy_jbar * ubar \
+                 + 1.0 / 4.0 * eth_j_dy_jbar * ubar) \
+                + one_minus_y \
+                  * (du_r_divided_by_r * dy_j * 1.0 / 4.0 \
+                       * (-2.0 * dy_jbar + jbar * dy_j_jbar / k_squared) \
+                     - 1.0/4.0 * dy_j * dy_jbar * w \
+                     + w * dy_j_jbar**2 / (16.0 * k_squared)) \
+                + one_minus_y**2 * (- dy_j * dy_jbar / (8.0 * r) \
+                                    + dy_j_jbar**2 / (32.0 * k_squared * r))
+
+    script_cj = 1.0 / 2.0 * ethbar_j * k * (eth_beta - 1.0 / 2.0 * q)
+
+    eth_j_qbar_minus_2_ethbar_beta = np.conj(ethbar_jbar_q_minus_2_eth_beta)
+    eth_qbar = np.conj(ethbar_q)
+    ethbar_r_divided_by_r = np.conj(eth_r_divided_by_r)
+    ethbar_ubar = np.conj(eth_u)
+    eth_ubar = np.conj(ethbar_u)
+    ubar = np.conj(u)
+    dy_jbar = np.conj(dy_j)
+
+    return -1.0 / 2.0 * eth_ubar_dy_j - 1.0 / 2.0 * u * ethbar_dy_j \
+           - 1.0 / 2.0 * u * dy_j * ethbar_r_divided_by_r \
+           + j * (script_bj + np.conj(script_bj)) \
+           + exp_2_beta / (2.0 * r) \
+             * (script_cj + j**2 / k_squared * np.conj(script_cj) \
+                + eth_eth_beta - 1.0 / 2.0 * eth_q \
+                - j * (script_aj + np.conj(script_aj)) \
+                + eth_j_qbar_minus_2_ethbar_beta / (4.0 * k) \
+                - j * eth_qbar / (4.0 * k) + (eth_beta - 1.0 / 2.0 * q)**2) \
+           - dy_j \
+             * (jbar * eth_u / (2.0 * k) - 1.0 / 2.0 * ethbar_ubar * j * k \
+                + 1.0 / 4.0 * (eth_ubar - ethbar_u) * k_squared + \
+                + 1.0 / 2.0 * eth_r_divided_by_r * ubar - 1.0 / 2.0 * w) \
+           + dy_jbar \
+             * (-1.0 / 4.0 * j**2 * (- eth_ubar + ethbar_u) \
+                + eth_u * j * (j * jbar /( 2.0 * k))) \
+           + one_minus_y \
+             * (1.0 / 2.0 * (-dy_j / r + 2.0 * du_r_divided_by_r * dy_dy_j \
+                             + w * dy_dy_j) + dy_j * (1.0 / 2.0 * dy_w \
+                                                      + 1.0 / (2.0 * r))) \
+           + one_minus_y**2 * dy_dy_j / (4.0 * r)
+
+
+def compute_linear_operator_for_h(script_dj, dy_j, j, one_minus_y):
```

Comment:
I think this is `linear_factor` now (there may be more occurrences where you used `operator` synonymously with `factor` before making the naming consistent, I haven't checked)